src/pkg/packaging/rpm/scripts/after_install_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "Creating dotnet host symbolic link: /usr/bin/dotnet"
+ln -sf "/usr/share/dotnet/dotnet" "/usr/bin/dotnet"
```

Comment:
I suppose this is a limitation of fpm, but generally the symlink itself would be a part of the package itself. That allows `rpm -qf /usr/bin/dotnet` to say that it's owned by the same package that owns `/usr/share/dotnet/dotnet`.